### DISCUSSION TITLE
Able to set different respawn timers for different vehicle layers.

### DISCUSSION THREAD

**Discussion by TheLocalPub** [Vdauphin/HeartsAndMinds#1604] [Question and answer] [answered]
Currently, there only one layer `btc_veh_respawnable` which anything placed inside will respawn after 30 seconds.
I've changed this timer to 300 seconds, but was looking to make other vehicles respawn, but on a separate timer, 600 second for example.
Is there a way of doing this currently, or will I have to go through the functions and add calls for a separate layer?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
You can just define a name to your vehicle and in init_server.sqf:
```sqf
[tlp_respawn_1, 600] call btc_veh_fnc_addRespawn;
```
Cheers

**Reply by TheLocalPub** [reply]
After asking this last night, I opt'd to use the vanilla respawn module.
I felt like it was much closer to what I wanted, with having options to despawn if no players are within X distance.
I also didn't like/want vehicles to deduce tickets, so instead removed them all from the respawn layer.
